physx/src/physics.rs

```diff
@@ -711,9 +735,18 @@ impl<Allocator: AllocatorCallback> PhysicsFoundationBuilder<Allocator> {
         self
     }
 
+    pub fn with_error_callback<EC: ErrorCallback>(&mut self, error_callback: EC) -> &mut Self {
+        self.error_callback = unsafe { Some(error_callback.into_px()) };
```

Comment:
Add todo about deallocating here so we don't forget?

Reply:
Created an issue.